tests/.snapshots/success.txt

```diff
@@ -22,25 +22,29 @@
   ✓ it throws exception if dataset already exist
   ✓ it sets closures
   ✓ it sets arrays
-  ✓ it gets bound to test case object with ('a')
-  ✓ it gets bound to test case object with ('b')
-  ✓ it truncates the description with ('FoooFoooFoooFoooFoooFoooFoooF...ooFooo')
-  ✓ lazy datasets with (1)
-  ✓ lazy datasets with (2)
+  ✓ it gets bound to test case object #1 with ('a')
```

Comment:
Can we just add the `#1` and those, only for cases that are duplicated? Example:
```
  ✓ lazy datasets with (1)
  ✓ lazy datasets with (2) #1
  ✓ lazy datasets with (2) #2
```

Reply:
There is another bug in my implementation. Numbering now destroys the order of descriptions. Need to check again, will ping you when I'm finished.

Reply:
@nunomaduro the bug is now fixed and also included in the test.